src/platform.rei

```diff
@@ -1,11 +1,19 @@
+type iosIdiom =
+  | Phone
+  | Pad
+  | TV;
+
 type os =
-  | IOS
-  | Android;
+  | IOS(iosIdiom)
+  | Android
+  | Web
```

Comment:
I generally don't think it's a good idea to mix `react-native-web` into the mix here because a number of little differences might make a few apis not as nice to use as they possibly could be.

That said, I do think we should provide escape hatches. IMO the best way to proceed is to remove web and sketch variants but leave `Unknown` as is.

---

EDIT: Alternatively both `os` and `version` could be changed to functions and throw exceptions in case of something unknown. The problem is that pattern matches should be exhaustive and the vast majority of apps uses one codebase only for the mobile. Therefore additional pattern matches for unused platforms might be a bit tedious for those apps.